content/docker-hub/repos/access.md

```diff
@@ -18,7 +18,7 @@ repository from that repository's **Settings** page.
 > **Note**
 >
 > A collaborator can't add other collaborators. Only the owner of
-> the repository has administrative access.
+> the repository has administrative access. Also, organizations can't add collaborators. Organization owners can control repository access with member roles and teams. See [Roles and permissions](../../security/for-admins/roles-and-permissions.md).
```

Comment:
`Also, organizations can't add collaborators.`
I got hung up on this. Maybe the missing part is "organization repositories".
Something like, organization repositories can't have collaborators. Or, you can't add collaborators to organization repositories.

Reply:
Makes sense, will update!